<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SphericalHarmonicsLightingClass</class>
 <widget class="QMainWindow" name="SphericalHarmonicsLightingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1529</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SphericalHarmonicsLighting</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridlayout" columnstretch="4,0,0">
    <item row="0" column="0">
     <widget class="Widget" name="openGLWidget"/>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout" columnstretch="3,0">
      <item row="7" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSlider_meshScale">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_contrast">
        <property name="text">
         <string>Contrast (2.0)</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_meshScale">
        <property name="text">
         <string>Mesh Scale</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLineEdit" name="lineEdit_refractRatio"/>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSlider_brightness">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_meshScale">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEdit_contrast">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSlider_refractRatio">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QComboBox" name="comboBox_materialType">
        <item>
         <property name="text">
          <string>Diffuse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Diffuse_SphericalHarmonic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mirror</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Glass</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_brightness">
        <property name="text">
         <string>Brightness (0.13)</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_refractRatio">
        <property name="text">
         <string>Refract Ratio</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit_ambientFactor">
        <property name="inputMask">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="lineEdit_brightness">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_materialType"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_materialType">
        <property name="text">
         <string>Material Type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Control Panel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_loadMesh">
        <property name="text">
         <string>Load Mesh</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSlider_ambientFactor">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_ambientFactor">
        <property name="text">
         <string>Ambient Factor (0.35)</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSlider_contrast">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1529</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Widget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">Widget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SphericalHarmonicsLighting.qrc"/>
 </resources>
 <connections/>
</ui>
